src/common/name-syntax.k

```diff
@@ -2,15 +2,16 @@ module NAME-SYNTAX
 imports LIST-SYNTAX
 //todo: why does this pure syntax definition affect krun?
 //PackageName, AmbName and PackageorTypeName syntax doesn't cause problems
-//                     syntax PackageName ::= IdList [klabel('PackageName)]
-//                     syntax ExprName::= Id  [klabel('ExprName)]
+                     syntax PackageName ::= IdList [klabel('PackageName)]
+                     syntax AmbName ::= Id [klabel('AmbName)]
+                                      | AmbName "." Id [klabel('AmbName)]
+                     syntax PackageOrTypeName ::= Id [klabel('PackageOrTypeName)]
+                                                | PackageOrTypeName "." Id [klabel('PackageOrTypeName)]//strict(1)
+syntax KLabel ::= "'MethodName"
```

Comment:
I cannot put it there as klabel attribute of syntax MethodName, the error info was not solvable by myself. Actually this name-syntax is where I showed you that syntax can break the program and where I asked Radu to look into it but didn't solve some time ago. These definitions strech the parser too much maybe. What I did was kind of using trial and error to maximize what valid syntax I could write there, and to be used by other productions. I will discuss this file again with Radu and see what solution we can come up with.